apps/remote_control/lib/lexical/remote_control/code_intelligence/definition.ex

```diff
@@ -62,6 +63,7 @@ defmodule Lexical.RemoteControl.CodeIntelligence.Definition do
             [entry]
         end
       end)
+      |> Enum.uniq_by(& &1.path)
```

Comment:
won't `uniq_by(& &1.subject)` also take care of it?

Reply:
> why's the sort before the uniq_by? Shouldn't you be sorting all the results?

I later realized that this was unnecessary, but I forgot to push the code yesterday.

> won't uniq_by(& &1.subject) also take care of it? 
Indeed, this is clearer.